test/infrastructure/docker/Dockerfile

```diff
@@ -48,14 +48,10 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg/mod \
     CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o /workspace/manager main.go
 
-# Use alpine:latest as minimal base image to package the manager binary and its dependencies
-FROM alpine:latest
+FROM golang:1.15.3
```

Comment:
why golang instead of plain ubuntu/debian? at this stage the go build is already completed...

Reply:
Mostly to avoid pulling yet another image during the build process

Reply:
I noticed that the main branch now has `golang:1.15.7` as the base image for the Dockerfiles. So we should probably update this.